src/inference/src/dev/icompiled_model_wrapper.cpp

```diff
@@ -10,7 +10,7 @@
 
 InferenceEngine::ICompiledModelWrapper::ICompiledModelWrapper(
     const std::shared_ptr<InferenceEngine::IExecutableNetworkInternal>& model)
-    : ov::ICompiledModel(nullptr, ov::legacy_convert::convert_plugin(model->_plugin)),
+    : ov::ICompiledModel(nullptr, ov::legacy_convert::convert_plugin(model->_plugin), nullptr, nullptr),
```

Comment:
why?

Reply:
this is not needed in wrapper right? won't they be released when compiled model destructed?

Reply:
Yes we don't need to create these task and callback executors, because new API just wraps old objects